website/docs/r/l3_ext_subnet.html.markdown

```diff
@@ -9,6 +9,8 @@ description: |-
 # aci_l3_ext_subnet #
 Manages ACI l3 extension subnet
 
+Tenant -> Networking -> L3Outs -> External EPGs -> Subnets -> Route Control Profile
+
```

Comment:
Add this information in the correct format. If you can add the rest it would be appreciated:
```
## API Information ##

* `Class` - vmmCtrlrP
* `Distinguished Named` - uni/vmmp-{vendor}/dom-{name}/ctrlr-{name}

## GUI Information ##

* `Location` - Virtual Networking -> {vendor} -> {domain_name} -> {controller_name}
```
This is an example from vmm_controller to give you a base